bloomstack_core/services/drivers.py

```diff
@@ -1,5 +1,5 @@
 import frappe
-from bloomstack_core.hook_events.delivery_trip import get_address_display
+from erpnext.stock.doctype.delivery_trip.delivery_trip import get_address_display
```

Comment:
```suggestion
from erpnext.stock.doctype.delivery_trip.delivery_trip import get_address_display_for_trip
```

also change this where the function is called